target POD remains in Error state after migration cancellation on crio

/kind bug

**What happened**:
In some cases, destination virt-launcher panics after migration cancellation, on CRIO. 
```
{"component":"virt-launcher","level":"info","msg":"stopping cmd server","pos":"server.go:313","timestamp":"2019-05-03T01:37:03.743387Z"}                                                                          
{"component":"virt-launcher","level":"error","msg":"Connection to libvirt lost","pos":"libvirt.go:257","timestamp":"2019-05-03T01:37:07.607429Z"}                                                                 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x519680]

goroutine 23 [running]:
kubevirt.io/kubevirt/vendor/github.com/libvirt/libvirt-go.(*Connect).IsAlive(0x0, 0x1638220, 0xc00018b3b0, 0x1)                                                                                                   
        connect.go:568 +0x40
kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli.(*LibvirtConnection).installWatchdog.func1(0xc0002b4420, 0x2540be400)                                                                                         
        pkg/virt-launcher/virtwrap/cli/libvirt.go:248 +0xff
created by kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli.(*LibvirtConnection).installWatchdog
        pkg/virt-launcher/virtwrap/cli/libvirt.go:238 +0x49
```

--- a/pkg/virt-launcher/virtwrap/cli/libvirt.go
+++ b/pkg/virt-launcher/virtwrap/cli/libvirt.go
@@ -122,7 +122,11 @@ func (l *LibvirtConnection) NewStream(flags libvirt.StreamFlags) (Stream, error)
 
 func (l *LibvirtConnection) Close() (int, error) {
 	close(l.stop)
-	return l.Connect.Close()
+	if l.Connect != nil {
+		return l.Connect.Close()
+	} else {
+		return 0, nil
+	}
 }
 
 func (l *LibvirtConnection) DomainEventLifecycleRegister(callback libvirt.DomainEventLifecycleCallback) (err error) {
@@ -243,9 +247,12 @@ func (l *LibvirtConnection) installWatchdog(checkInterval time.Duration) {
 				return
 
 			case <-time.After(checkInterval):
-				l.reconnectIfNecessary()
-
-				alive, err := l.Connect.IsAlive()
+				var alive bool
+				var err error
+				err = l.reconnectIfNecessary()
+				if l.Connect != nil {
+					alive, err = l.Connect.IsAlive()
+				}
 
 				// If the connection is ok, continue
 				if alive {

